examples/locality-aware-routing/datacenter/variables.tf

```diff
@@ -49,7 +49,7 @@ variable "consul_server_startup_timeout" {
 variable "consul_image" {
   description = "Consul Docker image."
   type        = string
-  default     = "hashicorppreview/consul-enterprise:1.17-dev"
+  default     = "hashicorp/consul-enterprise:1.18.0-rc1-ent"
```

Comment:
Do we need ent version here?

Reply:
Yes https://hub.docker.com/layers/hashicorp/consul-enterprise/1.18.0-rc1-ent/images/sha256-404aed724980f2ebcfba1bda04696c3156cca3767571711e213ae2ae4876d990?context=explore